cas-server-core/src/main/java/org/jasig/cas/logout/LogoutManagerImpl.java

```diff
@@ -150,12 +150,14 @@ public void setIssueAsynchronousCallbacks(final boolean asyncCallbacks) {
     /**
      * Service supports back channel single logout?
      * Service must be found in the registry. enabled and logout type must not be {@link LogoutType#NONE}.
-     * @param registeredService the service
+     * @param registeredService the registered service
```

Comment:
this still seems weird as the "registeredService" should have been the normalized version of all possible instances/representations of that specific service (i.e. it normalizes /MYAPP/page1 and /MYAPP/page2 to registeredService /MYAPP)